src/pc/tracker.rs

```diff
@@ -147,6 +147,8 @@ pub struct SubxtTracker {
 	disputes: Vec<DisputesTracker>,
 	/// Current relay chain block timestamp.
 	current_relay_block_ts: Option<u64>,
+	/// Current finality lag
+	current_finality_lag: Option<u32>,
```

Comment:
Since we don't keep track of prev value, we should just name it `finality_lag`
```suggestion
	/// Last observed finality lag
	finality_lag: Option<u32>,
```

Reply:
Changed